data_diff/dbt.py

```diff
@@ -220,16 +222,28 @@ def _local_diff(diff_vars: DiffVars) -> None:
 
 
 def _cloud_diff(diff_vars: DiffVars) -> None:
+    datafold_host = os.environ.get("DATAFOLD_HOST")
+    if datafold_host is None:
+        datafold_host = "https://app.datafold.com"
+    datafold_host = datafold_host.rstrip("/")
+    rich.print(f"Cloud datafold host: {datafold_host}")
```

Comment:
Not sure about printing this every time. Might make more sense as a debug log

Reply:
I add it just in case in order to a user can see that the correct host is used.